hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -693,6 +694,14 @@ message SwitchExceedThrottleQuotaResponse {
   required bool previous_exceed_throttle_quota_enabled = 1;
 }
 
+message BalancerDecisionRequest {
```

Comment:
Probably should be BalancerDecisionsRequest, but just a nit (usually there will be more than one decision, right?)